lib/worker.js

```diff
@@ -164,7 +164,7 @@ Worker.prototype._start = function() {
         self.serviceReturns = res;
         // Make sure that only JSON-serializable values are returned.
         try {
-            return [JSON.parse(JSON.stringify(res))];
+            return JSON.parse(JSON.stringify(res));
```

Comment:
Right, but we are getting an array of arrays in the master, as we wrapped it in the worker.